search-app/src/cmr/search/api/routes.clj

```diff
@@ -365,6 +376,11 @@
                {params :params headers :headers ctx :request-context}
                (get-deleted-collections ctx path-w-extension params headers)))
 
+        (context ["/:path-w-extension" :path-w-extension #"(?:collections-created-after)(?:\..+)?"] [path-w-extension]
```

Comment:
I don't think we need to create a new endpoint for this. User can just submit a search  with parameter `created-at` just like any other search parameter. All the `created-at` specific query functions that are added in this PR can then be removed.